.devcontainer/requirements.txt

```diff
@@ -15,4 +15,6 @@ pathvalidate==3.2.0
 svgpathtools==1.6.1
```

Comment:
the PR title should be ok, I already changed the autorisations last week to specifically avoid this error :thinking: 
Maybe it's because the PR is older. Could you re-open a clean new PR with just the river part ?
Thanks
![image](https://github.com/user-attachments/assets/d07665d2-52c5-418c-855d-a758fa160f11)

Reply:
Yes I am gonna do this

Reply:
@ThomasParistech  see https://github.com/ThomasParistech/pretty-gpx/pull/17